corehq/apps/user_importer/importer.py

```diff
@@ -821,6 +824,11 @@ def _modify_existing_user_in_domain(self, membership, role_qualified_id,
 
         imported_is_active = cv['is_active_in_domain']
         if str(current_user.is_active_in_domain(self.domain)) != imported_is_active:
+            record_event(WebUserRow.USER_IMPORTER_USER_ENABLED_STATUS_CHANGED, self.importer.upload_user, {
```

Comment:
Why make the name a constant rather than define it in-line?  Follow-up question - any performance concerns with doing this in bulk?  Like what if they inactivate a ton of users all at once?  Does `record_event` batch metric submissions?  Is there a way to do so?